smrf/tests/test_data_gridded.py

```diff
@@ -10,21 +9,19 @@
 
 class TestLoadGrid(SMRFTestCase):
 
-    def compare_hrrr_gold(self, out_dir):
+    def compare_hrrr_gold(self):
```

Comment:
Could this be put on the base class? We have 2 methods for comparing gold files, this one and the `compare_netcdf_files` in the `SMRFTestCase`